doc/architecture/frontend.md

```diff
@@ -0,0 +1,57 @@
+# Frontend Architecture
+
+## Structure
+
+- `__mocks__/`: Contains mocked modules for tests
+- `assets/`: Contains asset files like images
+- `bindings/`: Contains the generated API bindings
+- `components/`: Contains components that are shared across features
+- `config/`: Contains initialization code for the environment and libraries
+- `features/`: Contains the distinct features of the application
```

Comment:
Here some guideline what a "distinct features" is, and what the alternatives for implementations are.